packages/js/e2e-core-tests/core-tests-root.js

```diff
@@ -20,7 +20,8 @@ const getCoreTestsRoot = () => {
 
 	return {
 		appRoot: coreTestsPath,
-		packageRoot: moduleDir,
+		moduleRoot: moduleDir,
+		coreTestsRoot: __dirname
```

Comment:
I'll explain below that this PR changes the value of `appRoot`. `coreTestsRoot` contains the value that used to be `appRoot`. I'll do some research after submitting the review to discuss how we handle this. The monorepo reorg means I don't have access to the `blame` to see what `appRoot` is used for.

Reply:
@Luc45 Just to follow up- `appRoot` isn't used any more. I should have removed it in https://github.com/woocommerce/woocommerce/pull/31221 but I didn't check to see if it was used anywhere else.